src/reducers/index.ts

```diff
@@ -1,3 +1,16 @@
-const rootReducer = (state, _) => state
+const initialState = {
+    token: '',
+}
+
+const rootReducer = (state = initialState, action) => {
```

Comment:
shall we define a type for every action ?

Reply:
yes! you can take a look at https://github.com/HugoGresse/hacs-repositories/blob/master/src/packages/types.ts
Basicaly, the reducer take a typed action, every action has it's specific side, which allow you to get the specific type in the reducer itself within the switch. 
Also, in another PR we could introduce immer js for the reducer, easy to setup and work with.